src/setup_node_env/babel_register/register.js

```diff
@@ -19,12 +19,6 @@
 
 var resolve = require('path').resolve;
 
-// this must happen before `require('@babel/register')` and can't be changed
-// once the module has been loaded
-if (!process.env.BABEL_CACHE_PATH) {
-  process.env.BABEL_CACHE_PATH = resolve(__dirname, '../../../optimize/.babelcache.json');
```

Comment:
This cache path is used for server code, and is useful at least in development, so if we want to stop shipping the cache that's fine but if we could base it on `IS_KIBANA_DISTRIBUTABLE` that'd be better.

Reply:
@spalger as we talked offline, I miss the potential start time benefits of it as I was mainly focused in the optimization time. Let's keep it for now. I've also changed the name to `babel_register_cache.json` 😄